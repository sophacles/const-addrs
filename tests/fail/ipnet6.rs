use const_addrs::ipnet6;

fn main() {
    let _ = ipnet6!("2001:db8::32::23/129");
    let _ = ipnet6!("2001:db8::32:23/129");
}
