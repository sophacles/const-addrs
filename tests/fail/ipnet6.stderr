error: invalid IP address syntax
 --> tests/fail/ipnet6.rs:4:21
  |
4 |     let _ = ipnet6!("2001:db8::32::23/129");
  |                     ^^^^^^^^^^^^^^^^^^^^^^

error: invalid IP address syntax
 --> tests/fail/ipnet6.rs:5:21
  |
5 |     let _ = ipnet6!("2001:db8::32:23/129");
  |                     ^^^^^^^^^^^^^^^^^^^^^
